src/pages/Profile.tsx

```diff
@@ -1,38 +1,62 @@
 import React from "react";
 import { useEffect } from "react";
 import KeyboardBackspaceIcon from "@mui/icons-material/KeyboardBackspace";
-import { Button, Typography, Tabs, Tab, Avatar, Box } from "@mui/material";
+import {
+  Button,
+  Typography,
+  Tabs,
+  Tab,
+  Avatar,
+  Box,
+  Divider,
+  Link,
+} from "@mui/material";
 import EditIcon from "@mui/icons-material/Edit";
-import ProfileTweets from "../components/Profile/ProfileTweets";
+import ProfilePosts from "../components/Profile/ProfilePosts";
 import axios from "axios";
-import { useAppSelector } from "../state/hooks";
 import ProfileReplies from "../components/Profile/ProfileReplies";
 import ProfileLikes from "../components/Profile/ProfileLikes";
 import CalendarMonthIcon from "@mui/icons-material/CalendarMonth";
-import { useNavigate } from "react-router-dom";
+import { useNavigate, useParams } from "react-router-dom";
 import IconButton from "@mui/material/IconButton/IconButton";
 import Layout from "./Layout";
 import SideBar from "../components/SideBar/SideBar";
+import { Link as Routerlink } from "react-router-dom";
+import { useAppSelector } from "../state/hooks";
+import FollowButton from "../components/Common/FollowButton";
 
 const styles = {
   avatar: {
```

Comment:
I like the border change! Can we turn the height and width back to what it was before? The issue you probably ran into is that the `Avatar` component styles differently so you have to add `px` after the number. We should also make the box sizing border-box since we're adding a border:

```suggestion
  avatar: {
    border: "5px solid white",
    boxSizing: "border-box",
    height: "140px",
    marginTop: "-15%",
    width: "140px"
  },
```

Reply:
Implemented!